### PR TITLE
Fixed #31736 -- Fixed InspectDBTransactionalTests.test_foreign_data_wrapper crash on Windows.

### PR DESCRIPTION
**Pull request by lothemar**
Changed filename used in test_foreign_data_wrapper from constant '/dev/null' to os.devnull to work on Windows

### CODE REVIEW

File: tests/inspectdb/tests.py [hunk 1]
```diff
@@ -381,9 +382,9 @@ def test_foreign_data_wrapper(self):
                     sepal_length real,
                     sepal_width real
                 ) SERVER inspectdb_server OPTIONS (
-                    filename '/dev/null'
+                    filename '{}'
```
Comment (felixxm) [inline tests/inspectdb/tests.py:385]:
```suggestion
                    filename %s
```

File: tests/inspectdb/tests.py [hunk 2]
```diff
@@ -381,9 +382,9 @@ def test_foreign_data_wrapper(self):
...
                 )
-            ''')
+            '''.format(devnull))
```
Comment (felixxm) [inline tests/inspectdb/tests.py:387]:
```suggestion
            ''', [os.devnull])
```

File: tests/inspectdb/tests.py (hunk 1 above)
Comment (lothemar) [reply tests/inspectdb/tests.py:385]: Shouldn't we wrap the %s inside a string? Might be wrong here

**Comment by felixxm** [comment]
@lothemar Thanks :+1:

**Comment by lothemar** [comment]
Thank you @felixxm for the suggestions!

**Pull request merged**